### PR TITLE
feat(mobile-vitals): Add more mobile vitals to discover

### PR DESCRIPTION
**Pull request by Zylphrex**
This adds the following mobile vitals to discover

- `measurements.frames_slow_rate`
- `measurements.frames_frozen_rate`
- `measurements.stall_count`
- `measurements.stall_total_time`
- `measurements.stall_longest_time`
- `measurements.stall_rate`

Also allows users to use them in aggregate functions.

Depends on #27544

### CODE REVIEW

**Review by k-fish — APPROVED** [review]
To confirm, these will only show up in discover if you have the feature flag enabled right?

File: static/app/utils/performance/vitals/constants.tsx [hunk 1]
```diff
@@ -116,4 +116,52 @@ export const MOBILE_VITAL_DETAILS: Record<MobileVital, Vital> = {
...
+    slug: 'stall_longest_time',
+    name: t('Stall Longest Time'),
+    description: t(
+      'Stall Longest Time is the longest amount of time the application is stalled within a transaction.'
+    ),
+    type: measurementType(MobileVital.StallLongestTime),
+  },
+  [MobileVital.StallRate]: {
+    slug: 'stall_rate',
+    name: t('Stall Rate'),
```
Comment (k-fish) [inline static/app/utils/performance/vitals/constants.tsx:161]: I'm not sure stall time / transaction.duration is actually a `rate`.

It's more of a stalled_percentage maybe?

Comment (Zylphrex) [reply static/app/utils/performance/vitals/constants.tsx:161]: will update to `stalled_percentage` 👍

File: static/app/utils/performance/vitals/constants.tsx [hunk 2]
```diff
@@ -116,4 +116,52 @@ export const MOBILE_VITAL_DETAILS: Record<MobileVital, Vital> = {
...
+    slug: 'stall_total_time',
+    name: t('Stall Total Time'),
+    description: t(
+      'Stall Total Time is the total amount of time the application is stalled within a transaction.'
+    ),
+    type: measurementType(MobileVital.StallTotalTime),
+  },
+  [MobileVital.StallLongestTime]: {
+    slug: 'stall_longest_time',
+    name: t('Stall Longest Time'),
```
Comment (dashed) [inline static/app/utils/performance/vitals/constants.tsx:153]: "Longest Stall Time" seems to roll off the tongue better than "Stall Longest Time"

File: static/app/utils/performance/vitals/constants.tsx [hunk 3]
```diff
@@ -116,4 +116,52 @@ export const MOBILE_VITAL_DETAILS: Record<MobileVital, Vital> = {
...
+    slug: 'stall_count',
+    name: t('Stalls'),
+    description: t(
+      'Stalls is the number of times the application stalled within a transaction.'
+    ),
+    type: measurementType(MobileVital.StallCount),
+  },
+  [MobileVital.StallTotalTime]: {
+    slug: 'stall_total_time',
+    name: t('Stall Total Time'),
```
Comment (dashed) [inline static/app/utils/performance/vitals/constants.tsx:145]: Maybe "Total Stall Time", to mirror similarly to "Total Blocking Time"?

**Review by dashed — APPROVED** [review]

**Pull request merged**